ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main-periodics.yaml

```diff
@@ -993,6 +993,17 @@ periodics:
     job-release: "4.9"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-knative-serverless-operator-main-4.9-e2e-aws-ocp-49-continuous
+  reporter_config:
+    slack:
+      channel: '#serverless-dev'
+      job_states_to_report:
+      - success
```

Comment:
Do you want to log success? AFAIK, the other jobs in serverless operator only report on failure/error.

Reply:
It's inconsistent currently it seems, since I've copuied this.

Reply:
IMO, the weekly jobs (Azure,vSphere) log on success too but the regular nightly jobs log on failure.

Reply:
The weekly jobs do not run so often so it's good to see when it passes. But for nightly jobs there are several messages per day which are a little noisy.

Reply:
fair enough!